src/sonic-yang-models/yang-models/sonic-default-lossless-buffer-parameter.yang

```diff
@@ -0,0 +1,53 @@
+module sonic-default-lossless-buffer-parameter {
+
+    yang-version 1.1;
+
+    namespace  "http://github.com/Azure/sonic-default-lossless-buffer-parameter";
+
+    prefix default-lossless-buffer-parameter;
+
+    description "DEFAULT_LOSSLESS_BUFFER_PARAMETER YANG module for SONiC OS";
+
+    revision 2022-05-31 {
+        description "Initial version";
+    }
+
+    container sonic-default-lossless-buffer-parameter {
+        container DEFAULT_LOSSLESS_BUFFER_PARAMETER {
+
+            description "DEFAULT_LOSSLESS_BUFFER_PARAMETER part of config_db.json";
+
+            list DEFAULT_LOSSLESS_BUFFER_PARAMETER_LIST {
+                key "name";
+
+                leaf name {
+                    type string {
+                        pattern '[a-zA-Z0-9]{1}([-a-zA-Z0-9_]{0,31})' {
```

Comment:
Do we restrict the name in code with this pattern today? If not I prefer not to introduce any new patterns here as it cause breaks with already configured names which don't align with this pattern or length